.github/ISSUE_TEMPLATE/focus-area-proposal.yml

```
@@ -0,0 +1,63 @@
+name: Focus Area Proposal
+description: Proposal for an Interop 2026 Focus Area
+labels: focus-area-proposal
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to make a proposal for Interop 2026!
+
+      Be sure to read the [proposal guide](https://github.com/web-platform-tests/interop/blob/main/proposal_guide.md) to give your proposal the best chance of success!
+
+      Note: If this feature is widely implemented, but lacks a high quality specification or tests, an [Investigation Effort](https://github.com/web-platform-tests/interop/issues/new?template=investigation-effort-proposal.yml) may be more appropriate.
+
+      When you submit your proposal, a GitHub issue will be created to track it. Please check back on GitHub from time to time to answer any questions we have. You'll also be able to make changes to your proposal from now until October 2nd by commenting on your issue. See our [README](https://github.com/web-platform-tests/interop/blob/main/2026/README.md) for the full timeline of Interop 2026.
+- type: markdown
+  attributes:
+    value: "## Proposal Details"
+- type: textarea
+  attributes:
+    label: Description
+    description: Brief description of the feature. Try to be clear about what parts of the feature are included and not.
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Specification
+    description: |
+      Specification for the feature. The feature must be defined by a standards-track specification from one of the following organizations:
+
+      - [Alliance for Open Media](https://aomedia.org/)
+      - [IETF](https://www.ietf.org/) (Standard Track)
+      - [ISO](https://www.iso.org/)
+      - [Khronos Group](https://www.khronos.org/)
+      - [TC39](https://tc39.es/) (Stage 2+)
+      - [W3C](https://www.w3.org/) (Recommendation Track)
+      - [WebAssembly](https://webassembly.org/specs/) (Phase 3+)
+      - [WHATWG](https://whatwg.org/)
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: web-feature
+    description: |
+      [web-feature](https://web-platform-dx.github.io/web-features-explorer/) for the proposal. If your proposal corresponds closely to a specific web-feature please provide it here.
+- type: textarea
+  attributes:
+    label: Test Links
+    description: |
+      Links to web-platform-tests, if you already know which tests cover the proposal
```

```suggestion
      Links to web-platform-tests, if you already know which tests cover the proposal.
```
(nit to make punctuation consistent with other field descriptions)